### ISSUE TITLE
No menu bar on Windows RStudio 1.2v

### ISSUE THREAD

**Issue by Fredo-XVII** [rstudio/rstudio#4733]
### System details

    RStudio Edition : Desktop 
    RStudio Version : 1.2.1335
    OS Version      : Windows 10 Enterprise
    R Version       : 3.5.1

### Steps to reproduce the problem
Installed RStudio 1.2.1335

### Describe the problem in detail
There is no menu bar on windows.  I cannot set global options for this new RStudio version. Issue #2542 sounds similar but the description didn't quite convince me that it is the same issue.

**Comment by gtritchie** [comment]
Could you add a screenshot of what you are seeing? Thanks!

**Comment by Huh** [comment]
Same issue, I downloaded (version below), installed and opened.

- OS Windows 10 Pro
- Version 10.0.17763 Build 17763
- RStudio Download https://download1.rstudio.org/desktop/windows/RStudio-1.2.1335.exe
- Can provide RStudio diagnostics output if desired, but it contains several private bits, so will require cleaning

The issue is the top of the screen, the menu bar is hidden from view and the script that was opened by default cannot be closed, clicked on or otherwise manipulated now that I have opened a second script. The appearance is rather fuzzy as well, not sure how else to describe the appearance.

![screenshot_windows10](https://user-images.githubusercontent.com/483064/57342733-8318fb00-70fd-11e9-9cfd-919b3bc225c5.png)

**Comment by Huh** [comment]
This issue was not present on a new machine I just took out of the box. 

Steps taken, remove machine from box, install RStudio, install R 3.6.0, open RStudio. At this point RStudio asked if it should auto detect the rendering engine or if I wanted to choose. A notification then popped up and said that the rendering engine had been changed. Everything works fine on the new machine. I went back to the old machine and attempted to access the global options to play with the rendering engine option, opened it with alt + F and navigated to global options. At this point the options opened, but I was not allowed to click on much of anything anything and could not open the advanced tab. Several of the checkboxes were inaccessible as well. This is far beyond just not showing the menu bar, the whole tool is unusable at this point. I am reverting back to an old version on the old machine and look forward to using the new version on the new machine.

Hope that is helpful, I am willing to try things if you would like on the old machine to get it to work or help diagnose.

Settings and build were the exact same as my previous comment above.
- OS Windows 10 Pro
- Version 10.0.17763 Build 17763
- RStudio Download https://download1.rstudio.org/desktop/windows/RStudio-1.2.1335.exe
- Downloaded on 5/8/2019 at 1300 MT

**Comment by kevinushey** [comment]
You likely need to enable software rendering on that machine, or update your video card drivers. See https://support.rstudio.com/hc/en-us/articles/360017886674-Troubleshooting-RStudio-Rendering-Errors for more details.

**Comment by Fredo-XVII** [comment]
@huh, thanks for the reprex! I had to uninstall 1.2, and re install the old version in order to get work done.

**Comment by ronblum** [comment]
@huh and @Fredo-XVII What happens if you press <kbd>Ctrl</kbd> while starting RStudio, and change to software rendering that way? The support article above has a screenshot of the pop-up that should come up.

**Comment by Huh** [comment]
@ronblum I reinstalled the latest version of RStudio. When I open it while holding the Ctrl button I get the menu and have three options 1) Auto 2) Desktop OpenGL and 3) Software. I first opened RStudio by selecting the Desktop OpenGL rendering engine option, same result, no menu bar. I then closed RStudio and opened it again, but this time selected Software as the rendering engine and everything appeared as it should.
